examples/gloo/geometry_shader.py

```diff
@@ -7,6 +7,8 @@
 Use a Geometry Shader
 =====================
 
+NOTE: This example is currently not processed in CI.
```

Comment:
I'm 99% sure this will become the hover text of this example in the gallery. What do you think about making it a normal `#` comment below the docstring? Or adding an additional description paragraph above this. I think if this is the second paragraph of the description it isn't shown in the hover text but is shown in the gallery page.

Reply:
I did not check this, but you are probably right. I've copied this over from the other example which is ignored. But that has another two lines above that note.

I'll not be able to push changes atm, due to travelling. If this can wait some days, I'll fix this up at the end of the week.